gradle/publishing.gradle

```diff
@@ -13,6 +10,44 @@ publishing {
             artifact source: shadowJar
             artifact source: sourcesJar
             artifact source: javadocJar
+
+            pom {
+                name = 'NeonBee Core'
+                description = 'NeonBee is an open source reactive dataflow engine, a data stream processing framework using Vert.x.'
+                url = 'https://neonbee.io/'
+
+                licenses {
```

Comment:
I saw this information is redundant in the files quite often, isn't there a way to specify this only once?

Reply:
I don't know how. :(